go.work

```diff
@@ -0,0 +1,12 @@
+go 1.20
```

Comment:
I would remove these. We aren't using go.work files as there are problems with it.  I do not remember exactly what the problems were but the team has discussed it before and decided not to use it.